Coding-Coach/coding-coach
# Simpler solution for Panel components tailwind css

# Details

## Description
Removes the calcheight plugin, instead uses flexbox to create the Panel component.   

## Steps to QA
Load my PR code and https://codingcoach.io/ at the same time and compare the Panel component UI.
**Important Note** You will notice some changes like button is not aligned, and the 'Last updated' will be different sizes. 
* The button is different because it has not been converted to tailwind yet so the new css is overwriting it.  (I did a minor test to make sure that this was what was happening, switching some of the button css did resolve the issue but thats not what this PR is for)
* The 'Last updated' change when it was changed to markup.  Now, instead of a `small` tag, it is being represented as an `h4` (not sure if this is meets accessibility standards because it skips from `h2` to `h4`), but we will need to discuss how that should be resolved separately. 

**Side note**
* Changed both `h1` in the markdown to `h2` as they were before. 

## Screenshots
<img width="789" alt="screen shot 2018-12-07 at 7 47 16 pm" src="https://user-images.githubusercontent.com/22843948/49680335-f5a38300-fa58-11e8-80d6-839430e31de7.png">
<img width="789" alt="screen shot 2018-12-07 at 7 47 03 pm" src="https://user-images.githubusercontent.com/22843948/49680336-f5a38300-fa58-11e8-99f6-87011e133fc7.png">
<img width="789" alt="screen shot 2018-12-07 at 7 46 47 pm" src="https://user-images.githubusercontent.com/22843948/49680337-f5a38300-fa58-11e8-898e-a932958f2d89.png">
